feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb

```diff
@@ -0,0 +1,356 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -U feathr pandavro scikit-learn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisite: Configure the required environment\n",
+    "\n",
+    "In the first step (Provision cloud resources), you should have provisioned all the required cloud resources. If you use Feathr CLI to create a workspace, you should have a folder with a file called `feathr_config.yaml` in it with all the required configurations. Otherwise, update the configuration below.\n",
+    "\n",
+    "The code below will write this configuration string to a temporary location and load it to Feathr. Please still refer to [feathr_config.yaml](https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml) and use that as the source of truth. It should also have more explanations on the meaning of each variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tempfile\n",
+    "yaml_config = \"\"\"\n",
+    "# Please refer to https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml for explanations on the meaning of each field.\n",
+    "api_version: 1\n",
+    "project_config:\n",
+    "  project_name: 'feathr_home_credit'\n",
+    "  required_environment_variables:\n",
+    "    - 'REDIS_PASSWORD'\n",
+    "    - 'AZURE_CLIENT_ID'\n",
+    "    - 'AZURE_TENANT_ID'\n",
+    "    - 'AZURE_CLIENT_SECRET'\n",
+    "offline_store:\n",
+    "  adls:\n",
+    "    adls_enabled: tru\n",
+    "  wasb:\n",
+    "    wasb_enabled: true\n",
+    "  s3:\n",
+    "    s3_enabled: false\n",
+    "    s3_endpoint: 's3.amazonaws.com'\n",
+    "  jdbc:\n",
+    "    jdbc_enabled: false\n",
+    "    jdbc_database: 'feathrtestdb'\n",
+    "    jdbc_table: 'feathrtesttable'\n",
+    "  snowflake:\n",
+    "    url: \"dqllago-ol19457.snowflakecomputing.com\"\n",
+    "    user: \"feathrintegration\"\n",
+    "    role: \"ACCOUNTADMIN\"\n",
+    "spark_config:\n",
+    "  spark_cluster: 'azure_synapse'\n",
+    "  spark_result_output_parts: '1'\n",
+    "  azure_synapse:\n",
+    "    dev_url: \"https://feathrhomecreditcaspark.dev.azuresynapse.net\"\n",
+    "    pool_name: \"spark31\"\n",
+    "    # workspace dir for storing all the required configuration files and the jar resources\n",
+    "    workspace_dir: \"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/\"\n",
+    "    executor_size: \"Small\"\n",
+    "    executor_num: 4\n",
+    "    feathr_runtime_location: wasbs://public@azurefeathrstorage.blob.core.windows.net/feathr-assembly-LATEST.jar\n",
+    "  databricks:\n",
+    "    workspace_instance_url: 'https://adb-6885802458123232.12.azuredatabricks.net/'\n",
```

Comment:
This too and to its effect any hardcoded URLs

Reply:
Removed the databricks configuration as it is not needed in this example.